src/sst/outgoing_request_bridge.hh

```diff
@@ -97,8 +109,18 @@ class OutgoingRequestBridge: public SimObject
     // Returns the buffered data for initialization. This is necessary as
     // when gem5 sends functional requests to memory for initialization,
     // the connection in SST Memory Hierarchy has not been constructed yet.
+    // This buffer is only used during the INIT phase.
     std::vector<std::pair<Addr, std::vector<uint8_t>>> getInitData() const;
 
+    // We need Set/Get functions to set the init_phase_bool.
+    // `initPhaseComplete` is used to signal the outgoing bridge that INIT
+    // phase is completed and RUN phase will start.
+    void initPhaseComplete(bool value);
+
+    // We read the value of the init_phase_bool using `getInitPhaseStatus`
+    // method.
+
```

Comment:
This should not be a newline... overall, the style is not up to the guidelines of gem5's style guide.

Reply:
ive added a new newline. i'll update the file with smaller patches to adhere with the gem5's style.